packages/util/src/array.spec.ts

```diff
@@ -120,3 +122,43 @@ describe('array', () => {
     });
   });
 });
+
+describe('zip', () => {
+  it('zips two arrays where left is longer than right', () => {
+    expect(zip([1, 3, 5], [2, 4])).toEqual([1, 2, 3, 4, 5]);
+  });
```

Comment:
perhaps also a type test that you can not mix different types with zip.

Reply:
Zip and most other array functions use a generic type. Testing this seems trivial to me and would rather check the correctness of generics in TypeScript than our code in my opinion. Or do you see other issues that I didn't?

Reply:
no I also thought about the triviality and it is clear from the method signature/definition.